### ISSUE TITLE
Mopidy can't play playlist from SoundCloud

### ISSUE THREAD

**Issue by fata1224** [mopidy/mopidy-soundcloud#41]
Hello everyone,

I am currently using a Raspberry Pi to run a MusicBox Pi.
However, when I try to play a playlist from SoundCloud, it doesn't work.
ex: I input the following setting ( sc:https://soundcloud.com/yuhsiang/sets/music ) in the 'Streams' page of MusicBox MPD.

Is there one can help?

**Comment by JC3** [comment]
Did you ever solve this?

**Comment by dreamlayers** [comment]
Using Mopidy 3.0.1 and the most recent mopidy-soundcloud, both from the Mopidy package repository, I was able to add `sc:https://soundcloud.com/liquidsounddesign/sets/saafibrothers_mysticremixes` to the tracklist via Mopidy-Mobile web interface. When I added that, all the individual tracks in that were added to the tracklist, and they are playable. So, apparently this works and the issue should probably be closed.
